helix-term/src/commands/lsp.rs

```diff
@@ -64,28 +64,83 @@ fn jump_to_location(
     align_view(doc, view, Align::Center);
 }
 
+/// Returns a string showing the symbol nested in its containing symbol if it exists.
+///
+/// # Arguments
+/// * `symbol` - the symbol that we're creating the string for
+///
+/// # Example
+/// If you have something like:
+/// ```
+/// mod foo {
+///     fn bar() {}
+/// }
+/// ```
+/// the result will be: `function  : foo → bar`
+fn get_symbol_string(symbol: &lsp::SymbolInformation) -> String {
+    let kind = match symbol.kind {
+        lsp::SymbolKind::FILE => "file      ",
+        lsp::SymbolKind::MODULE => "module    ",
+        lsp::SymbolKind::NAMESPACE => "namespace ",
+        lsp::SymbolKind::PACKAGE => "package   ",
+        lsp::SymbolKind::CLASS => "class     ",
+        lsp::SymbolKind::METHOD => "method    ",
+        lsp::SymbolKind::PROPERTY => "property  ",
+        lsp::SymbolKind::FIELD => "field     ",
+        lsp::SymbolKind::CONSTRUCTOR => "construct ",
+        lsp::SymbolKind::ENUM => "enum      ",
+        lsp::SymbolKind::INTERFACE => "interface ",
+        lsp::SymbolKind::FUNCTION => "function  ",
+        lsp::SymbolKind::VARIABLE => "variable  ",
+        lsp::SymbolKind::CONSTANT => "constant  ",
+        lsp::SymbolKind::STRING => "string    ",
+        lsp::SymbolKind::NUMBER => "number    ",
+        lsp::SymbolKind::BOOLEAN => "boolean   ",
+        lsp::SymbolKind::ARRAY => "array     ",
+        lsp::SymbolKind::OBJECT => "object    ",
+        lsp::SymbolKind::KEY => "key       ",
+        lsp::SymbolKind::NULL => "null      ",
+        lsp::SymbolKind::ENUM_MEMBER => "enum mem  ",
+        lsp::SymbolKind::STRUCT => "struct    ",
+        lsp::SymbolKind::EVENT => "event     ",
+        lsp::SymbolKind::OPERATOR => "operator  ",
+        lsp::SymbolKind::TYPE_PARAMETER => "type param",
```

Comment:
I think even if you want to align, it is better to remove the spaces, then use left align format specifier to do the spacing rather than having to do that manually. Doing it manually will cause changing the largest variant require changing all the lines, bad for git diff.